vpc/globalgc/globalgc.go

```diff
@@ -115,20 +116,21 @@ func fetchDisconnectedENIs(parentCtx *context.VPCContext, vpcID string, networkI
 // TODO: Find fatal errors and return errors
 func cleanupENI(parentCtx *context.VPCContext, svc *ec2.EC2, detachTime time.Duration, eni *ec2.NetworkInterface) error {
 	ctx := parentCtx.WithField("NetworkInterfaceId", *eni.NetworkInterfaceId)
+	logger := ctx.Logger.WithField("eni-name", *eni.NetworkInterfaceId)
 	tags := ec2util.TagSetToMap(eni.TagSet)
 	networkCreationTime, ok := tags[vpc.ENICreationTimeTag]
 	if !ok {
-		ctx.Logger.Warning("ENI does not have creation time field")
+		logger.Warning("ENI does not have creation time field")
```

Comment:
why not use the context logger?

Reply:
No reason - was just trying to pick one of the loggers - this function was using both `parentCtx.Logger` and `ctx.Logger` before.  Based on your comment above, using `ctx.Logger` makes more sense.